config/bundles.php

```diff
@@ -1,16 +1,5 @@
 <?php
 
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-declare(strict_types=1);
```

Comment:
Can you please, bring this content back?

Reply:
Sorry, I have restored.